src/core/public/chrome/chrome_service.test.ts

```diff
@@ -228,7 +230,7 @@ describe('start', () => {
 
       await expect(promise).resolves.toMatchInlineSnapshot(`
         Array [
-          true,
+          false,
```

Comment:
snapshots changed due to the change of logic of the initial `isVisible$` value computation